### PR TITLE
Updated Codingstyle-pom revApi config

### PR DESCRIPTION
**Pull request by DJCoding**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -135,6 +136,17 @@
...
+      <dependency>
+        <groupId>org.revapi</groupId>
+        <artifactId>revapi-java</artifactId>
+        <version>${revapi-java.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.revapi</groupId>
+        <artifactId>revapi-reporter-json</artifactId>
+        <version>${revapi-reporter-json-version}</version>
+      </dependency>
```
Comment (uhafner) [inline pom.xml:149]: Why is that needed? The versions given in `build` will override these values.

**Review by uhafner — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (DJCoding) [reply pom.xml:149]: I forgot to talk with you about this topic. Is it possible to declare the java and reporter dependency in this pom so that we don't need to declare it again in every child pom? Would be even better for that analysis pom which get used quite often as parent.

File: pom.xml [hunk 2]
```diff
@@ -700,7 +723,7 @@
             <execution>
               <id>run-revapi</id>
               <goals>
-                <goal>check</goal>
+                <goal>report</goal>
```
Comment (uhafner) [inline pom.xml:726]: This should use check as well here.

**Review by uhafner — APPROVED** [review]
